sample/src/androidTest/java/com/schibsted/spain/barista/sample/ScrollsTest.java

```diff
@@ -23,36 +23,58 @@
   public FailureHandlerValidatorRule handlerValidator = new FailureHandlerValidatorRule();
 
   @Test
-  public void scrollsInsideScrollable() {
+  public void scrollsInsideScrollable_byId() {
     assertTopVisible();
-
     scrollTo(R.id.really_far_away_button);
+    assertBottomVisible();
+  }
 
+  @Test
+  public void scrollsInsideScrollable_byHardcodedText() {
+    assertTopVisible();
+    scrollTo("Really far away button");
+    assertBottomVisible();
+  }
+
+  @Test
+  public void scrollsInsideScrollable_byTextResource() {
+    assertTopVisible();
+    scrollTo(R.string.really_far_away_button);
     assertBottomVisible();
   }
 
   @Test(expected = BaristaException.class)
   public void scrollsOutsideScrollableFails() throws Exception {
     assertTopVisible();
-
     scrollTo(R.id.centered_button);
+    assertTopVisible();
```

Comment:
This is not really wanted. Because `scrollTo` is the method that should fail.

If by any chance the `scrollTo` doesn't fail (like safelyScrollTo) BUT `assertTopVisible` throws a BaristaException, the test will be green while the behavior is broken.

Reply:
Wow, totally missed the expected when doing the merge :·/ Thanks for spotting!